R/later_recurring.R

```diff
@@ -0,0 +1,24 @@
+#' @describeIn later Schedules a recurring task
+#' @param limit Number of times to repeat the function. If `NA` (the default)
+#'   then no limit.
+#' @examples
+#' later_recurring(~cat("Hello from the past\n"), 3, limit = 2)
+#' @export
+later_recurring <- function(func, delay, limit = NA, loop = current_loop()) {
+  func <- rlang::as_function(func)
+  cancelled <- FALSE
+  if (!is.na(limit) && limit < 1)
+    stop("'limit' must be 'NA' or a positive number")
```

Comment:
I suggest

```r
if (is.na(limit)) limit <- Inf
```

since it catches both `NA` and `NaN`. Is there a reason `%|%` (no `NaN` protection) is still better?

Reply:
Works for me! 
I think it was an artifact of some earlier changes I had made but didn't keep. `if (is.na(limit)) limit <- Inf` is _cleaner_